addons/mail/static/src/core/web/mail_core_web_service.js

```diff
@@ -30,6 +30,31 @@ export class MailCoreWeb {
                 this.store.starred.counter--;
             }
         });
+        this.busService.subscribe("mail.record/update_activity", async (payload) => {
```

Comment:
I think that it should/can be done as part of a `mail.record/insert` notification, same for the `delete_activity` one.